lib/read-stream.js

```diff
@@ -143,15 +156,17 @@ class ReadStream extends stream.Readable {
       this._readBlock()
     }
 
-  }
+  },
 
   destroy() {
     fs.close( this.fd, ( error ) => {
       this.fd = null
       stream.Readable.prototype.destroy.call( this )
     })
-  }
+  },
 
 }
 
+inherit( ReadStream, stream.Readable )
```

Comment:
But it seems to be [used](https://www.npmjs.com/browse/depended/bloodline) only by your own projects?
(I don't mean any disrespect, but I wonder if there's a more common / popular / well known module that does something similar?)
I'm not that familiar with npm or nodejs (or inheritance in JS), so please take my comments with a large pinch of salt :-)